tests/platform_tests/test_reload_config.py

```diff
@@ -157,9 +165,14 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     result, out = execute_config_reload_cmd(duthost)
     # config reload command shouldn't work immediately after system reboot
     assert result and "Retry later" in out['stdout']
-
     assert wait_until(300, 20, 0, config_system_checks_passed, duthost, delayed_services)
 
+    # Check if all containers have started
+    assert wait_until(300, 10, 0, check_docker_status, duthost)
```

Comment:
Does this check compatible with all kinds of physical testbeds?

Reply:
Sure.